README.md

```diff
@@ -1,12 +1,67 @@
 # CosmJS
 
-This is a JavaScript/TypeScript client-side binding to [wasmd](https://github.com/cosmwasm/wasmd), a sample blockchain for the [cosmwasm](https://github.com/confio/cosmwasm) smart contracting platform.
+CosmJS is the swiss army knife to power JavaScript based client solutions
+ranging from Web apps/explorers over browser extensions to server-side clients
+like faucets/scrapers in the Cosmos ecosystem.
+
+"Cosm" stands for Cosmos and "JS" stands for _runs everywhere_ – we actually
```

Comment:
Suggest "is short for" instead of "stands for".

Reply:
Thanks. What about the second "stands for"? Is this correct or a stange German translation? "is short for" probably does not work for the second case, right?

Reply:
Sounds natural to me 👍 .